### DISCUSSION TITLE
[v5] adonis-hashids package

### DISCUSSION THREAD

**Discussion by vicainelli** [adonisjs/core#1936] [Help]
Hey all, did someone know if [adonis-hashids](https://github.com/carlsonorozco/adonis-hashids) is compatible with the AdonisJS v5? 
And if yes, how to set it up...

**Comment by RomainLanz** [comment]
Hey @vicainelli! 👋 

It doesn't seem to have been updated for V5.

You can directly use the package [hashids](https://www.npmjs.com/package/hashids) within your application if you want to "hash" your IDs.

**Reply by dawidow** [reply]
@RomainLanz hey bro, do you have any plans to update your [adonis-guard](https://github.com/RomainLanz/adonis-guard) package in the near future?

**Reply by RomainLanz** [reply]
We plan to have that feature built-in.
You can follow the rfcs with the following link https://github.com/adonisjs/rfcs/issues/26
